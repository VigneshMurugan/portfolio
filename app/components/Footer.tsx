export interface FooterProps {}

export const Footer = (_props: FooterProps) => {
  const date = new Date();
  const year = date.getFullYear();

  return (
    <footer className="text-center text-sm">
      <div className="mb-2 flex items-center justify-center gap-4">
        <img
          alt="Follow me on GitHub"
          className="footer-social"
          height={16}
          src="/github.svg"
          width={16}
        />
        <img
          alt="Follow me on Twitter"
          className="footer-social"
          height={16}
          src="/twitter.svg"
          width={16}
        />
        <img
          alt="Follow me on LinkedIn"
          className="footer-social"
          height={16}
          src="/linkedin.svg"
          width={16}
        />
      </div>
      <p>
        Built with <span className="text-color-primary">&hearts;</span> in{" "}
        <a
          // href="https://www.google.com/search?q=san+diego+weather"
          href="https://www.surfline.com/surf-report/carlsbad-state-beach-north/5d4dc10680c2430001b119fb"
          rel="noreferrer"
          target="_blank"
        >
          San Diego
        </a>
        , California. &copy; {year}
      </p>
    </footer>
  );
};
