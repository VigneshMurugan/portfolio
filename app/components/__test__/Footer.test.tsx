import { expect, test } from "vitest";
import { render } from "@testing-library/react";

import { Footer } from "../Footer";

describe("Footer Component", () => {
  const component = render(<Footer />);

  test("should render our footer", () => {
    expect(component.baseElement).toMatchSnapshot();
  });
});

test("should work as expected", () => {
  expect(Math.sqrt(4)).toBe(2);
});
