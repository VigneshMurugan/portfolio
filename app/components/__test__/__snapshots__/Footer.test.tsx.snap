// Vitest Snapshot v1

exports[`Footer Component > should render our footer 1`] = `
<body>
  <div>
    <footer
      class="text-center text-sm"
    >
      <p>
        Built with 
        <span
          class="text-color-primary--"
        >
          ♥
        </span>
         in
         
        <a
          href="https://www.google.com/search?q=san+diego+weather"
          rel="noreferrer"
          target="_blank"
        >
          San Diego
        </a>
        , California. © 
        2022
      </p>
    </footer>
  </div>
</body>
`;
